config/updates/8005/field.storage.paragraph.field_oe_icon.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
```

Comment:
we need to do this also for new sites, so we need to update this config in the install folder